Bottom: b27186addca21db11a0adf843c58867fef1d8b5f
Top:    d21e300b575581e5d8e4530b1f27f00456f65b63
Author: Peter Clifton <pcjc2@cam.ac.uk>
Date:   2011-04-20 22:02:02 +0100

Refresh of consolidate-pin-drawing-routin

---

diff --git a/src/crosshair.c b/src/crosshair.c
index 0ab6753..f016094 100644
--- a/src/crosshair.c
+++ b/src/crosshair.c
@@ -249,7 +249,7 @@ XORDrawElement (ElementTypePtr Element, LocationType DX, LocationType DY)
     PinType moved_pin = *pin;
     moved_pin.X += DX; moved_pin.Y += DY;
 
-    gui->thindraw_pcb_pin (Crosshair.GC, &moved_pin, false, false);
+    gui->thindraw_pcb_pv (Crosshair.GC, Crosshair.GC, &moved_pin, false, false);
   }
   END_LOOP;
